docs/deploying.md

```diff
@@ -481,6 +481,25 @@ OpenStackDataPlaneNodeSet has the following status fields:
 | DNSClusterAddresses | |
 | CtlplaneSearchDomain | |
 
+##### OpenStackDataPlaneNodeSet Pods
+
+Pods within OpenStackDataPlaneNodeSet can fail for many reasons.
+Under normal circumstances, these failures may be transient.
+However, for facilitating triage, the operator does check on service pods and reports
+if their status is indicative of failure.
+
+Failure of individual pods within nodeset is logged on info level in format:
+
+ `"openstack dataplane pod <POD_NAME> failed due to <FAILURE_REASON> message: <FAILURE_MSG>"`
```

Comment:
I think we should change this log message in the code as well, for all the above reasons.

Reply:
What kind of message would you prefer?

Reply:
Message changed.